sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/errorhandling/BadRecordRouter.java

```diff
@@ -65,6 +76,31 @@ public <RecordT> void route(
         throw new RuntimeException("Encountered Bad Record: " + encodedRecord);
       }
     }
+
+    @Override
+    public <RecordT> void route(
```

Comment:
neither c nor window is used here; outputReceiver also not used in the other implementation. Consider create a private method that both route calling this one?

Reply:
good call, cleaned up